### ISSUE TITLE
[Bug] Application freeze/crash on Chrome when expected arguments are not present

### ISSUE THREAD

**Issue by scotmatson** [graphql/graphiql#567]
When I have a query which receives arguments and refresh the browser, the arguments passed into the `Query Variables` window are flushed and the query is automatically rerun. This behavior results in a bypass of the error handling mechanism which checks to see if variables have actually been passed and results in a seemingly infinite loop cycle in which the entire GraphiQL application seizes up.

When this behavior occurs I am unable to interact with the window at all and must close the browser tab. I don't know if given enough time I will regain control of the window.. frankly I'm impatient and in most cases it is faster to just reconstruct my query. For complex queries however this is somewhat of a nuisance.

tl:dr
1) Open GraphiQL, insert query/mutation that receives query variables
2) Enter query variables into query variables window
3) Refresh browser tab or duplicate browser tab
  a. results in query variables being cleared
  b. GraphiQL freezes

Browser:

- Chrome: Version 60.0.3112.90 (Official Build) (64-bit)

Server: 

- click==6.7
- Flask==0.12.2
- Flask-Cors==3.0.2
- Flask-GraphQL==1.4.1
- graphene==1.4.1
- graphene-sqlalchemy==1.1.1
- graphql-core==1.1
- graphql-relay==0.4.5
- iso8601==0.1.12
- itsdangerous==0.24
- Jinja2==2.9.6
- MarkupSafe==1.0
- promise==2.0.2
- psycopg2==2.7.1
- PyJWT==1.4.2
- singledispatch==3.4.0.3
- six==1.10.0
- SQLAlchemy==1.1.13
- typing==3.6.1
- Werkzeug==0.12.2

**Comment by thomasheyenbrock** [comment]
@scotmatson do you have a link to a running reproduction for this? Is this still an issue with the latest version of GraphiQL (currently 1.8.10)?

**Comment by scotmatson** [comment]
Hi @thomasheyenbrock 

This is a few years old now and developing has not been my primary function for awhile. I'll go ahead and close this out as it is no longer something I would be able to reproduce.

**Comment by thomasheyenbrock** [comment]
Thanks for coming back to this and closing! 🙌
